locales/fr/translation.json

```diff
@@ -42,39 +42,39 @@
   "Account": "Compte",
   "Account Settings": "Paramètres de compte",
   "Activate Pod": "Activer le pod",
-  "All changes saved.": "Tous les changements ont été sauvegardés",
+  "All changes saved.": "Tous les changements ont été enregistrés",
   "Allow uploading": "Autoriser le téléchargement",
-  "All changes saved.": "Tous les changements ont été sauvegardés",
+  "All changes saved.": "Tous les changements ont été enregistrés",
   "Allow uploading": "Autoriser le téléchargement",
   "An error occurred while logging in.": "Une erreur s'est produite lors de la connexion",
-  "An error occurred while signing up.": "Une erreur s'est produite lors de la déconnexion",
-  "An unknown error occurred": "Une erreur inconnue est survenue ",
-  "and below": "et en dessous de ",
+  "An error occurred while signing up.": "Une erreur s'est produite lors de la crétaion du compte",
+  "An unknown error occurred": "Une erreur inconnue est survenue",
+  "and below": "et en dessous de",
   "Anonymous user": "Utilisateur anonyme",
-  "Anything you would like to share?": "Tout ce que vous voudriez partager",
-  "Approx {{numVisibleDays}} days in view": "Environ {{numVisibleDays}} jours de vue",
+  "Anything you would like to share?": "Que voulez-vous partager?",
+  "Approx {{numVisibleDays}} days in view": "Environ {{numVisibleDays}} jours",
   "April": "Avril",
-  "Are you sure you want to cancel your invitation to {{email}}?": "Êtes-vous sur de vouloir annuler votre invitation à",
-  "Are you sure you want to leave this person's Care Team? You will no longer be able to view their data.": "Êtes-vous sûr de vouloir quitter l'équipe soignante de cette personne? Vous ne pourrez plus voir leurs données.",
-  "Are you sure you want to remove this person? They will no longer be able to see or comment on your data.": "Êtes-vous sûr de vouloir supprimer cette personne? Ils ne pourront plus voir ou commenter vos données.",
-  "At least three days do not have boluses, so this statistic might not be right": "",
+  "Are you sure you want to cancel your invitation to {{email}}?": "Êtes-vous sur de vouloir annuler votre invitation à {{email}}?",
```

Comment:
sûr 

Et quand on aura du temps on essaiera de tout mettre en inclusif ( sûr•e )
;)

Reply:
c bien noté ;-)